library/VTKExtensions/Rendering/vtkF3DOpenGLGridMapper.h

```diff
@@ -58,7 +58,7 @@ class vtkF3DOpenGLGridMapper : public vtkOpenGLPolyDataMapper
 
   double FadeDistance = 10.0;
   double UnitSquare = 1.0;
```

Comment:
shoulnt the default be negative ?

Reply:
Technically it could be initialized to anything because it gets explicitly set later but making it non-positive here in `vtkF3DOpenGLGridMapper` would mean we have an invalid value by default which relies on an external piece of code to be made right (the automatic adjustment is outside the class in `vtkF3DRenderer.cxx`).

Reply:
good point.